cmd/version.go

```diff
@@ -39,6 +39,8 @@ var versionCmd = &cobra.Command{
 
 		fmt.Println("odo " + VERSION + " (" + GITCOMMIT + ")")
 
+		// turning off the flag which checks for login status
+		GlobalConnectionCheck = true
```

Comment:
I just realized that `GlobalConnectionCheck` is not correctly named.
Shouldn't it be called `GlobalSkipConnectionCheck`? As `true` means that there will be no connection check.

Reply:
done,  renamed